mutmut/__init__.py

```diff
@@ -22,6 +25,109 @@ def __eq__(self, other):
 
 ALL = MutationID(line='%all%', index=-1, line_number=-1)
 
+
+class ASTPattern(object):
+    def __init__(self, source, **definitions):
+        self.definitions = definitions
+        self.module, self.markers = _create_pattern_node(source, definitions=definitions)
+        self.pattern = [x['node'] for x in self.markers if x['marker'] == '↑'][0]
```

Comment:
True.. the obscure character is a leftover from an earlier draft where I wanted to be sure I didn't conflict with contents of the code, but that's irrelevant now.

Reply:
@ looks worse, it's less intuitive I think. Maybe instead of `↑` we just use `^` always and the pattern/matched node is picked based on the name `match` if it exists or if there is only one we pick that one?

Reply:
Would using two characters raise any issues? If not why not use `^M` for marker and `^V` for value?

Maybe just use `M` and `V`?

Reply:
Well, it't not really a value, it's just a pointing thing so you can refer to the node later. It's kind of like a capture group in regex. And I don't like V specifically because it visually points down while I want something that visually points towards the node I am interested in.